05-JWT-Basics/starter/errors/custom-error.js

```diff
@@ -1,7 +1,7 @@
 class CustomAPIError extends Error {
   constructor(message, statusCode) {
     super(message)
-    this.statusCode = statusCode
+    //this.statusCode = statusCode
```

Comment:
avoid committing commented-out code, prefer to delete it! In this case, we can also lose the now-empty constructor and leave it at:

```javascript
class CustomAPIError extends Error {};
```

It's still good to create `CustomAPIError`, because it creates a distinct error compared to the generic `Error` object.